doc/user/install-operator-sdk.md

```diff
@@ -70,10 +70,8 @@ $ brew install operator-sdk
 ## Compile and install from master
 
 ```sh
-$ mkdir -p $GOPATH/src/github.com/operator-framework
-$ cd $GOPATH/src/github.com/operator-framework
-$ git clone https://github.com/operator-framework/operator-sdk
-$ cd operator-sdk
+$ go get -d github.com/operator-framework/operator-sdk #This will download the git repository and not install it
```

Comment:
One more minor nit:
```suggestion
$ go get -d github.com/operator-framework/operator-sdk # This will download the git repository and not install it
```